source/includes/fact-save-pipeline.rst

```diff
@@ -0,0 +1,21 @@
+You can save a pipeline so that you can find it later. If you
+load a saved pipeline, you can change it without
+changing the original saved copy. You can also create a :ref:`view
+<view-details>` from your pipeline results.
+
+To save your pipeline:
+
+1. Click the :guilabel:`Save` dropdown button at the top of the pipeline
```

Comment:
Can we use `.. procedure::` here as well?

Reply:
Updated.